daemon/algod/api/algod.oas2.json

```diff
@@ -1075,6 +1075,36 @@
         }
       }
     },
+    "/v2/ready": {
+      "get": {
+        "tags": [
+          "public",
+          "nonparticipating"
```

Comment:
So participating nodes won't get the ready functionality?

Reply:
It's just a logical grouping that denotes this API isn't involved in participation. I've tried to describe this a bit better in the readme. https://github.com/algorand/go-algorand/blob/master/daemon/algod/api/README.md#adding-a-new-v2-api

Reply:
For completeness - With @bbroder-algo, I verbally discussed that it's worth considering if the endpoint ought to be exposed as a _common_ endpoint (not `v2/` prefixed) like `/health`. 

We didn't reach a conclusion during the verbal.  I expect @bbroder-algo + @ahangsu will evaluate the topic while working through the PR.

Reply:
The common endpoints are a bit different since they don't have their routes generated. The handlers and the routes use the same code as the v1 endpoints.

It would be good to get all of the common endpoints updated to use the v2 style. But yes I agree it seems like this should be a common API.

Reply:
moving to categories `public` and `common`, rather than staying in `v2`.